wire-ios-data-model/Source/Model/Conversation/ZMConversation+SecurityLevel.swift

```diff
@@ -424,7 +424,8 @@ extension ZMConversation {
         }
     }
 
-    private func discardPendingMessagesAfterPrivacyChanges() {
+    @objc(discardPendingMessagesAfterPrivacyChanges)
```

Comment:
suggestion: adding more details on this

```suggestion
    /// Discards all unsent messages since conversation's privacy changed.
    @objc(discardPendingMessagesAfterPrivacyChanges)
```